packages/reporter/src/std/StdReporter.ts

```diff
@@ -19,6 +20,7 @@ export class StdReporter implements Reporter {
     (...x: unknown[]) => void
   > = {
     /* eslint-disable no-console */
+    [Severity.CRITICAL]: console.error,
```

Comment:
Critical issues are not being reported due to the code at https://github.com/NeuraLegion/sectester-js/blob/bc05852b5252e71352a11e3f4e684f95641ab0f1/packages/reporter/src/std/StdReporter.ts#L34